mysql-test/suite/tokudb.rpl/include/rpl_tokudb_innodb.inc

```diff
@@ -0,0 +1,213 @@
+# Tests basic replication interoperability between tokudb and other storage
+# engines, specifically innodb.
+# Credit to Sveta Smirnova for the basic idea.
+connection slave;
+stop slave;
+start slave;
+connection master;
+
+-- disable_query_log
+set sql_log_bin=0;
```

Comment:
Great catch, I accidentally deleted the re-enabling of the binlog when refactoring the original case into a generic case to test both directions...killing this PR and will resubmit once fixed and retested.